cmake/BuildFlags.cmake

```diff
@@ -1,74 +1,81 @@
 # Setup building flags for SYCL device and host codes.
 
-# Support GCC only at the moment.
-if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
We still need keep the check for building on Linux. Since we only support GNU compiler so far. We will evaluate Clang compiler in future.

Reply:
Thanks, updated.